src/Exception/InvalidParameterException.php

```diff
@@ -12,7 +12,32 @@
 namespace Tobscure\JsonApi\Exception;
 
 use Exception;
+use Throwable;
```

Comment:
True, strange that the tests passed on older PHP versions - I'll remove the typehints though.